public/img/hide.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <path fill="#666" d="M9.79551332,17.5615051 L7.85697847,20.8684175 L6.13158253,19.8569785 L15.6430215,3.63158253 L17.3684175,4.64302153 L15.8255288,7.27500801 C17.6248861,8.45836775 19,10.2898022 19,12 C19,14.8335879 15.2249788,18 12,18 C11.2765756,18 10.5254736,17.8406691 9.79551332,17.5615051 Z M14.811603,9.00464621 L13.6863221,10.9242431 C13.8848965,11.2348665 14,11.6039963 14,12 C14,13.1045347 13.1045347,14 12,14 C11.9614665,14 11.9231874,13.9989101 11.885192,13.9967596 L10.8255411,15.8043989 C11.2237133,15.9295058 11.6211123,16 12,16 C14.1906056,16 17,13.6435363 17,12 C17,11.0410235 16.0435328,9.83932852 14.811603,9.00464621 Z M6.76979258,15.5815356 C5.70670597,14.51434 5,13.2260167 5,12 C5,9.16641209 8.77502116,6 12,6 C12.2396735,6 12.4823849,6.01748854 12.726359,6.05102931 L11.4827635,8.04078214 C9.39205786,8.35986392 7,10.4834429 7,12 C7,12.5675121 7.33496882,13.2200277 7.86376601,13.8311782 L6.76979258,15.5815356 Z"/>
```

Comment:
non-branded SVG files shouldn't have a `fill` set, only files that have a specific branding color.